Assets/Scripts/UserInput.cs

```diff
@@ -5,12 +5,20 @@
 public class UserInput : MonoBehaviour
 {
 
-    public GameObject slot1;
+    public GameObject selectedHandCard;
+
+    private PointSystem pointSys;
+
+    private string HAND_0 = "Hand0";
```

Comment:
Sorry, can you make these const as well and top of the page
```suggestion
    private const string HAND_0 = "Hand0";
```